src/main/java/com/autotune/analyzer/utils/AnalyzerConstants.java

```diff
@@ -241,117 +241,137 @@ public int getSeverity() {
     }
 
     public enum RecommendationNotification {
-        DURATION_BASED_RECOMMENDATIONS_AVAILABLE (
+        INFO_DURATION_BASED_RECOMMENDATIONS_AVAILABLE(
```

Comment:
As I had mentioned previously (in a different PR), we need to move all of these recommendation related constants out of `AnalyzerConstants` and instead inside the `recommendations` package